src/lava/proc/lif/models.py

```diff
@@ -107,7 +107,7 @@ def run_spk(self):
             # In Loihi, user specified threshold is just the mantissa, with a
             # constant exponent of 6
             self.effective_vth = np.left_shift(self.vth, self.vth_shift)
-            self.b_vth_computed = True
+            # self.b_vth_computed = True
```

Comment:
Delete the line if not used.

Reply:
This will go away with merge to the latest LIF in main